Formula/zsh.rb

```diff
@@ -10,6 +10,7 @@ class Zsh < Formula
     # We cannot build HTML doc on HEAD, because yodl which is required for
     # building zsh.texi is not available.
     option "with-texi2html", "Build HTML documentation"
+    option "with-nvcsformats-patch", "Apply patch that fixes nvcsformats for 5.2"
```

Comment:
nvm. But that would force my patch to be optional as it should be incompatible with `--head`, shouldn't it?